src/styles/style.css

```diff
@@ -0,0 +1,58 @@
+body {
```

Comment:
- [ ] OPTIONAL
Your application is not properly laid out in desktop versions. It appears at the bottom of the page. Kindly improve upon this aspect by ensuring that your app is accessible without scrolling the page.

**Your Application desktop view:**
![image](https://user-images.githubusercontent.com/12745474/134148559-1cfe4a77-4516-42bf-be0f-4a882f38c33b.png)